BlockchainSdk/Blockchains/Bitcoin/BitcoinWalletManager.swift

```diff
@@ -166,7 +172,7 @@ extension BitcoinWalletManager: BitcoinTransactionFeeCalculator {
 
 @available(iOS 13.0, *)
 extension BitcoinWalletManager: TransactionSender {
-    func send(_ transaction: Transaction, signer: TransactionSigner) -> AnyPublisher<TransactionSendResult, Error> {
+    func send(_ transaction: Transaction, signer: TransactionSigner) -> AnyPublisher<TransactionSendResult, SendTxError> {
```

Comment:
может тут вернуть 
```suggestion
    func send(_ transaction: Transaction, signer: TransactionSigner) -> some Publisher<TransactionSendResult, SendTxError> {
```

много `eraseToAnyPublisher` уйдет

Reply:
@Balashov152 положил в беклог, Этот мр и так разросся

https://tangem.atlassian.net/browse/IOS-6865